inspirehep/config.py

```diff
@@ -1437,7 +1438,7 @@ def _(x):
             },
             "source": {
                 "terms": {
-                    "field": "metadata.acquisition_source.method",
+                    "field": "metadata.acquisition_source.source",
```

Comment:
we could rename `submitter` to `forms` or something like that, but not sure if we gain anything.